From 17c0ac783c25dde8796f012cdec514d0b09f8f76 Mon Sep 17 00:00:00 2001
From: Mikel Astiz <mikel.astiz@bmw-carit.de>
Date: Fri, 31 Aug 2012 12:51:08 +0200
Subject: [PATCH 35/48] bluetooth: Do not switch to HFGW automatically

Card profile hfgw should be no different from the rest, and thus no
internal policy inside module-bluetooth-device should decide to switch
to its profile automatically.

This should be handled by policy modules.
---
 src/modules/bluetooth/module-bluetooth-device.c |   16 ----------------
 1 files changed, 0 insertions(+), 16 deletions(-)

diff --git a/src/modules/bluetooth/module-bluetooth-device.c b/src/modules/bluetooth/module-bluetooth-device.c
index ad68cfa..99a943b 100644
--- a/src/modules/bluetooth/module-bluetooth-device.c
+++ b/src/modules/bluetooth/module-bluetooth-device.c
@@ -1295,22 +1295,6 @@ static DBusHandlerResult filter_cb(DBusConnection *bus, DBusMessage *m, void *us
     } else if (dbus_message_is_signal(m, "org.bluez.HandsfreeGateway", "PropertyChanged")) {
         pa_bt_audio_state_t state = parse_state_property_change(m);
 
-        switch(state) {
-            case PA_BT_AUDIO_STATE_INVALID:
-            case PA_BT_AUDIO_STATE_DISCONNECTED:
-            case PA_BT_AUDIO_STATE_CONNECTED:
-            case PA_BT_AUDIO_STATE_CONNECTING:
-                goto fail;
-
-            case PA_BT_AUDIO_STATE_PLAYING:
-                if (u->card) {
-                    pa_log_debug("Changing profile to hfgw");
-                    if (pa_card_set_profile(u->card, "hfgw", FALSE) < 0)
-                        pa_log("Failed to change profile to hfgw");
-                }
-                break;
-        }
-
         if (state != PA_BT_AUDIO_STATE_INVALID) {
             pa_device_port *port;
             pa_port_available_t available = audio_state_to_availability(state);
-- 
1.7.7.6

